scripts/generate_docs.py

```diff
@@ -1,13 +1,13 @@
 """
- Copyright (C) 2023 Intel Corporation
+ Copyright (C) 2023-2024 Intel Corporation
  
  Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 """
 
 from pathlib import Path
 from shutil import rmtree
-import subprocess
+import subprocess   # nosec B404
```

Comment:
This one and `B603` could be placed in some bandit config file, if possible. It seems like they will always be needed when using `subprocess.run()`.

Reply:
I guess I didn't want to introduce "global" skips. In-place skips are only placed in dev scripts/tests, after looking into the issues.